app/models/miq_request_workflow.rb

```diff
@@ -33,6 +33,13 @@ def self.all_encrypted_options_fields
     descendants.flat_map(&:encrypted_options_fields).uniq
   end
 
+  def self.possibly_update_user(data, user)
+    return user if data[:user_name].blank?
+    new_user = User.lookup_by_identity(data[:user_name])
+    _log.warn "Web-service requester changed to <#{new_user.userid}>"
```

Comment:
This value can come from Rest or automate.  Now would be a good time to update the logging message.  Should probably just be `requested changed to <#{new_user.userid}>`